playground/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Can this be [inherited](https://www.typescriptlang.org/tsconfig#extends) from a base config up a directory?

Reply:
Probably